src/CreatePackage.php

```diff
@@ -0,0 +1,238 @@
+ * @copyright Copyright (c) 2017 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   https://github.com/zendframework/maintainers/blob/master/LICENSE.md New BSD License
+ */
+
+namespace ZF\Maintainer;
+
+use InvalidArgumentException;
+use RecursiveDirectoryIterator;
+use RecursiveIteratorIterator;
+use RuntimeException;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\ConsoleOutputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ChoiceQuestion;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+use Symfony\Component\Console\Question\Question;
+
+class CreatePackage extends Command
+{
+    protected function configure()
+    {
+        $this
+            ->setName('create-package')
+            ->setDescription('Creates a new package')
+            ->setHelp(
+                'The new package will be created in the directory of provided package name.'
+                . ' The package will be initialized with the whole basic structure.'
+                . ' It will be possible to choose namespace, based on provided package name,'
+                . ' or provide another namespace. At the end of the process it is possible to'
+                . ' create initial commit with the whole package structure.'
+                . PHP_EOL . PHP_EOL
+                . 'Package is created with documentation templates, all support files,'
+                . ' QA tools (CodeSniffer and PHPUnit), default ConfigProvider with tests,'
+                . ' Travis CI and coveralls configuration.'
+                . PHP_EOL . PHP_EOL
+                . 'Package name can be provided with the organization in format org/name.'
+                . ' When organization is not specified explicitly the script will try to'
+                . ' detect it based on the name. For names starting from "zend-" and "zf-"'
+                . ' organizations "zendframework" and "zfcampus" will be used respectively.'
+            )
+            ->addArgument(
+                'name',
+                InputArgument::REQUIRED,
+                'The name of the package to create; can be provided with organization name: org/name'
+            );
+    }
+
+    protected function initialize(InputInterface $input, OutputInterface $output)
+    {
+        parent::initialize($input, $output);
+
+        $name = $input->getArgument('name');
+        if (! preg_match('#^[a-z0-9-]+(/[a-z0-9-]+)?$#', $name)) {
+            throw new InvalidArgumentException(
+                'Invalid package name, can contain only lowercase letters, numbers and dash.'
+            );
+        }
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $name = $input->getArgument('name');
+        list($org, $repo) = $this->getOrg($name);
```

Comment:
If we make the requirement PHP 7.1, the left side can become `[$org, $repo]`. :smile:

(It's fine as-is, just pointing that out.)

Reply:
I know, but I still would like to support PHP 5.6. We can drop PHP 5.6 and point just to PHP 7.1 but I'd like to have it as many components still supports PHP 5.6